### ISSUE TITLE
In Ubuntu：ReferenceError: can't access lexical declaration `dialog' before initialization

### ISSUE THREAD

**Issue by just-tt** [siara-cc/sqlite3_page_explorer#3]
![图片](https://user-images.githubusercontent.com/33444186/66796098-37367180-ef39-11e9-8141-5d2b7668ac66.png)

when i click "open database",the issue has appear.

ReferenceError: require is not defined

**Comment by siara-cc** [comment]
Please try `v1.0-linux` release.  I tried it was working on Linux Mint.
What version of Ubuntu are you using?

**Comment by just-tt** [comment]
This is my Ubuntu.

![图片](https://user-images.githubusercontent.com/33444186/66976552-e0659f00-f0d4-11e9-9105-82714d886637.png)

v1.0-linux  Refers to the Kernel version?

**Comment by siara-cc** [comment]
No, `v1.0-linux` refers to a release on this repo. See https://github.com/siara-cc/sqlite3_page_explorer/releases.
If still does not work, you could clone this repo, do `npm update` and use the command `npm run-script package-linux32` or `npm run-script package-linux54` to generate executable for your platform, assuming you have `npm` and `electron` installed.

**Comment by siara-cc** [comment]
@just-tt  I have upgraded to latest electron so this issue should be fixed now.
Please see: https://github.com/siara-cc/sqlite3_page_explorer#getting-started
